<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget_fileShowcase">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>256</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>280</y>
      <width>401</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Section</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_sectionName"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_createSection">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/Images/Icons/icons8-approved-64.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_deleteSection">
       <property name="styleSheet">
        <string notr="true">
border-image: url(:/Images/Icons/icons8-cross-mark-button-48.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTreeWidget" name="treeWidget_keyValue">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>451</width>
      <height>241</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Key</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Value</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>320</y>
      <width>441</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Additing</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>171</width>
          <height>111</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Type</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Key</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Value</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QComboBox" name="comboBox_addType">
             <item>
              <property name="text">
               <string>Int</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Double</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Bool</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>String</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_keyAdditing"/>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_valueAdditing"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="pushButton_addKeyValue">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>80</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Deleting</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>151</width>
          <height>26</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Key</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_deleteKey"/>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="pushButton_deleteKey">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>80</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_openFile"/>
    <addaction name="separator"/>
    <addaction name="action_openDirectory"/>
    <addaction name="separator"/>
    <addaction name="action_saveFile"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_openFile"/>
   <addaction name="action_openDirectory"/>
   <addaction name="action_saveFile"/>
   <addaction name="actionCreate_File"/>
  </widget>
  <action name="action_openFile">
   <property name="icon">
    <iconset resource="Resourse.qrc">
     <normaloff>:/Images/Icons/icons8-opened-folder-100 (1).png</normaloff>:/Images/Icons/icons8-opened-folder-100 (1).png</iconset>
   </property>
   <property name="text">
    <string>Открыть файл</string>
   </property>
  </action>
  <action name="action_openDirectory">
   <property name="icon">
    <iconset resource="Resourse.qrc">
     <normaloff>:/Images/Icons/icons8-active-directory-48.png</normaloff>:/Images/Icons/icons8-active-directory-48.png</iconset>
   </property>
   <property name="text">
    <string>Открыть дирректорию</string>
   </property>
  </action>
  <action name="action_saveFile">
   <property name="icon">
    <iconset resource="Resourse.qrc">
     <normaloff>:/Images/Icons/icons8-save-file-64.png</normaloff>:/Images/Icons/icons8-save-file-64.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить файл</string>
   </property>
  </action>
  <action name="actionCreate_File">
   <property name="icon">
    <iconset resource="Resourse.qrc">
     <normaloff>:/Images/Icons/icons8-add-file-100.png</normaloff>:/Images/Icons/icons8-add-file-100.png</iconset>
   </property>
   <property name="text">
    <string>Create File</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resourse.qrc"/>
 </resources>
 <connections/>
</ui>
